content/en/docs/pipelines/tutorials/benchmark-examples.md

```diff
@@ -64,7 +64,7 @@ as an example.
 Use the following instructions to run the benchmark script on your Kubeflow
 Pipelines cluster.
 
-1. Download the the [run_service_api.ipynb](https://storage.googleapis.com/ml-pipeline/sample-benchmark/run_service_api.ipynb)
+1. Download the [run_service_api.ipynb](https://storage.googleapis.com/ml-pipeline/sample-benchmark/run_service_api.ipynb)
```

Comment:
If you have time, maybe we can also use "code font" for file names (e.g. lines 67 and others - `.ipynb` notebooks and YAML files (search for `.yaml`):

```suggestion
1. Download the [`run_service_api.ipynb`](https://storage.googleapis.com/ml-pipeline/sample-benchmark/run_service_api.ipynb)
```

(Reference - [Google docs style guide: Code in text](https://developers.google.com/style/code-in-text#some-specific-items-to-put-in-code-font)).